### PR TITLE
Create UI Track Playing Activity

### PR DESCRIPTION
**Pull request by nguyenvanducc**
https://edu-redmine.framgia.vn/issues/1561

### CODE REVIEW

**Comment by nguyenvanducc** [comment]
ready

File: app/src/main/java/com/framgia/nguyenvanducc/soundcloud33/screen/trackplaying/TrackPlayingActivity.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import android.support.v7.widget.Toolbar;
+import android.widget.TextView;
+
+import com.framgia.nguyenvanducc.soundcloud33.R;
+import com.framgia.nguyenvanducc.soundcloud33.screen.BaseActivity;
+
+public class TrackPlayingActivity extends BaseActivity {
+    private Toolbar mToolbar;
+    private TextView mTextToolbarTitle;
+    private TextView mTextToolbarSubtitle;
```
Comment (tuannguyen38) [inline app/src/main/java/com/framgia/nguyenvanducc/soundcloud33/screen/trackplaying/TrackPlayingActivity.java]: 11 12 13 cân nhắc là local variable hay không?

File: app/src/main/res/layout/activity_track_playing.xml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                android:layout_weight="2"
+                android:ellipsize="end"
+                android:maxLines="1"
+                android:textColor="@android:color/white"
+                android:textSize="@dimen/sp_14" />
+        </LinearLayout>
+    </android.support.v7.widget.Toolbar>
+
+    <android.support.v4.view.ViewPager
+        android:id="@+id/viewpager_playing_track"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/activity_track_playing.xml]: rename to view_pager_playing_track or view_pager_playing

File: app/src/main/res/layout/fragment_track_detail.xml [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+        android:layout_marginBottom="@dimen/dp_32"
+        android:layout_marginEnd="@dimen/dp_16"
+        android:layout_marginRight="@dimen/dp_16"
+        android:text="@string/default_time"
+        android:textColor="@android:color/white"
+        app:layout_constraintBottom_toTopOf="@+id/image_play"
+        app:layout_constraintEnd_toEndOf="parent" />
+
+    <SeekBar
+        android:id="@+id/seekBar"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/fragment_track_detail.xml]: covention 3.2

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -9,4 +9,5 @@
     <string name="playlist">Playlist</string>
```
Comment (tuannguyen38) [inline app/src/main/res/values/strings.xml]: trong strings.xm. em nên xem lại convention 3.2.2
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md

**Comment by nguyenvanducc** [comment]
ready

**Pull request merged**